src/tests/enforce-boolean-naming-prefixes.test.ts

```
@@ -0,0 +1,451 @@
+      return userExists(id);
+    }
+
+    // Using imported namespace
+    const result = utils.validate();
+
+    // Using imported class
+    const externalInstance = new ExternalClass();
+    const value = externalInstance.getValue();
+    `,
+
+    // Edge cases
```

ESLint

[error] Insert `··`

(prettier/prettier)